image_card.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
 /**
- * Plugin Name:       Wasyo Image Card
- * Description:       Wasyo custom image card for services
+ * Plugin Name:       Image Card
```

Comment:
This is wrong. The plugin must be named:

  Wasya Co Wp Plugin

You named it "Image Card". The plugin can provide the image card widget, but the name of the plugin must be none other than "Wasya Co Wp Plugin".